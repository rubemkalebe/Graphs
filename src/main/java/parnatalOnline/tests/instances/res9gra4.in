9 4
12 2 29 6 25 10 29 20
29 21 14 4 22 24 27
21 29 21 14 3 15
28 12 10 17 5
5 11 3 16
29 21 30
4 4
1
