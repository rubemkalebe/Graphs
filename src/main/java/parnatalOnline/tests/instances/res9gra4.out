27.000000
0 2, 1 5, 2 7, 3 8, 4 5, 4 7, 6 7, 7 8, 
4609080
5.194489;0.003998;